src/sidebar.jsx

```diff
@@ -315,7 +313,7 @@ const DropdownWithKebab = ({
             id: "edit-permissions",
             onClick: () => {
                 editPermissions(Dialogs, {
-                    selected: selected[0] || currentDirectory,
+                    selected: selected[0] || { name: directory_name, type: "dir" },
```

Comment:
this is still utter crap, but let's imagine a way to do that via UseContext. Gotta sit down and think about that, can we use `rootInfo`?